roles/ood_user_reg_cloud/tasks/main.yml

```diff
@@ -20,7 +20,7 @@
 - name: Change ownership of directory
```

Comment:
I'm thinking we should remove this task entirely.   It's actually better to have the application install owned by root (the installer of the app) and then have the application run as an unprivileged user which has no ability to change any of the installed application's files.